relay-base-schema/src/events.rs

```diff
@@ -71,6 +74,7 @@ impl FromStr for EventType {
             "expectct" => EventType::ExpectCt,
             "expectstaple" => EventType::ExpectStaple,
             "transaction" => EventType::Transaction,
+            "feedback" => EventType::UserReportV2,
```

Comment:
Similar question: If the string representation is `"feedback"`, why not call it `UserFeedback` in code?

Reply:
when discussing with @jan-auer over initial call, he called out to not have it so there are "synonyms" in the code. I can change to UserFeedback if that is preferred.